common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -64,24 +65,51 @@ div.video {
         }
       }
     }
+
     .video-tracks {
       > a {
         border-radius: 3px 0 0 3px;
       }
+
       > a.external-track {
         border-radius: 3px;
       }
     }
+
+    .branding {
+      display: inline-block;
+      float: right;
+      margin: 15px 0 0 10px;
+      vertical-align: top;
+
+      .host-tag {
+        // The @extend %cont-text-hide wasn't working for some reason...
```

Comment:
Apparently this extend doesn't exist, even though it does. Advised to just use overrides in this case. Will cleanup later after pattern library and cleanup starts.